Assets/GothicVR/Scripts/Player/Menu/TextureMenu.cs

```diff
@@ -11,7 +11,6 @@ public class TextureMenu : MonoBehaviour
         private void Start()
         {
             SetMaterials();
-            FontManager.I.ChangeFont();
```

Comment:
No need any longer? Did you implement Font loading in this branch?

Reply:
No need to set the font in script if we already use the empty font to force unity to use the spriteAsset containing gothic's font